utils/pdf/pdf.ts

```diff
@@ -575,13 +576,29 @@ export const getJournalContent = (entries: Array<OfficeJournalProps> = [{}, {},
   }
 }
 
-export const createPdf = (fileName: string, props: object) => {
+export const getGridContent = (title, grid: GridElement[][]) => {
+  return {
+    content: [
+      {text: title, style: 'title', margin: [0, 20]},
+      {
+        style: 'table',
+        table: {
+          // eslint-disable-next-line @typescript-eslint/no-unused-vars
+          widths: grid[0].map((_) => '*'),
+          heights: grid.map((_) => 25),
+          body: grid.map((r) => r.map((c) => c.value)),
+        },
+      },
+    ],
```

Comment:
If you add this here, you don't have to change `createPdf`'s signature. (and it makes more sense to enforce this at creation)
```suggestion
    ],
    pageOrientation: 'landscape',
```

Reply:
Good catch! Fixed in the next commit.